### PR TITLE
Add desktop-version to portfolio

### PR DESCRIPTION
**Pull request by luisrpavanelli**
Hello, I worked in this PR to add modifies in files html and css, and also add new background images

Desktop Version:
- Create a new branch
- Change html files
- Change css files
- Add backgrounds images

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -79,34 +82,47 @@ header {
...
 
-.maincard {
+.workscard {
   height: 474px;
-  margin: 25px;
-  justify-content: flex-end;
+  width: 369px;
+  left: 0;
+  top: 0;
+  border-radius: 8px;
```
Comment (mckent05) [inline style.css:94]: - This section of your page doesn't exactly look like the Figma design template, Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
-- The cards in this section should be aligned in a column of 3 and 2 rows. Please fix this 👍 👍 
-- For each of the cards, the image should come first before the other texts or details, please fix this 👍 
-- The text `My Recent Works` should be aligned in the center. Please fix this 😄 
ryjdryjryj
Figma template:
wefefe

File: style.css [hunk 2]
```diff
@@ -79,34 +82,47 @@ header {
...
-  justify-content: center;
-  margin: 12px 0;
+.aboutcard > * {
+  margin: 0;
+}
+
+.aboutcard .tags {
+  background-color: #d0d9d4;
+  margin: 5px;
+  padding: 12px;
```
Comment (mckent05) [inline style.css:122]: - This part of your portfolio does not look exactly like the given design template. Please fix this 😄 

Your project:
-- The section header i.e `About Me` text, section text and `Get Resume` button should be aligned in the center. Please fix this 🙏 
-- The background images for this section are missing, please fix this 🙏 
-- The technology cards are not designed the way they are in the Figma template, please include some spacing between the cards 👍 
eetrhethse

Figma template:
eargaerg

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @luisrpavanelli, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 


## Required Changes ♻️
- Your contact form is not designed as it is in the Figma design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏
Your project:
-- Please ensure all texts and form inputs in this section are aligned as they are in the figma template
-- Please ensure that the background images in this section are visible.
tnsthset
Figma design:
wwrvsdd


_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by luisrpavanelli** [comment]
Hello, I worked in this PR to add modifies in files HTML and CSS, and also add folders, new background images and icons...

Desktop Version:

- Create a new branch
- Modify HTML files Mobile to Desktop Version
- Modify CSS files Mobile to Desktop Version
- Add Backgrounds Images and Icons
- Add Folders Desktop Image and Desktop Icons

File: style.css [hunk 3]
```diff
@@ -229,81 +227,164 @@ hr.roundeddivider {
...
-  background-position: center;
+  padding: 64px 24px 26px;
+  background-image: url(./Desktop_Images/backgroundcontactform.svg);
   background-repeat: no-repeat;
-  background-size: cover;
-  padding: 0 24px;
-  display: flex;
-  flex-direction: column;
-  justify-content: flex-end;
-}
```
Comment (Shisui6) [inline style.css:241]: Great job redesigning the contact section, as suggested by the previous reviewer👍. However, you forgot to add the background image. Please add it, so it matches the figma template better🙏.

Your design:
image

Figma design:
image

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @luisrpavanelli, 

# Requires Changes 🍀
Great work on making the changes requested by a previous reviewer 👍
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet🪲.


 ## Highlights
 - Good design ✅
 - Good use of media queries ✅

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by luisrpavanelli** [comment]
Hello, I worked in this PR to add modifies in files HTML and CSS, and also add folders, new background images and icons...

Desktop Version:

Create a new branch
Modify HTML files Mobile to Desktop Version
Modify CSS files Mobile to Desktop Version
Add Backgrounds Images and Icons
Add Folders Desktop Image and Desktop Icons

Changes:

- Reduce the font size to match the figma template. 
- Add the background image to matches the figma template.

**Review by Zuhaib042 — APPROVED** [review]
Hi @luisrpavanelli , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
approve
## Highlights
✅ Correct Github flow
✅ Good implementation of suggested changes
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**